.circleci/config.yml

```diff
@@ -5,10 +5,13 @@ orbs:
   codecov: codecov/codecov@1.0.5
   browser-tools: circleci/browser-tools@0.1.3
 
+defaults: &defaults
```

Comment:
just curious what this means?

Reply:
Its a variable block, something you can reference later using the <<: *[name] syntax